lib/shopify_app/sessions_controller.rb

```diff
@@ -16,6 +16,7 @@ def callback
         sess = ShopifyAPI::Session.new(shop_name, response['credentials']['token'])
         session[:shopify] = ShopifyApp::SessionRepository.store(sess)
         session[:shopify_domain] = shop_name
+        session[:return_to] = nil
```

Comment:
err now `redirect_to return_address` won't work at all. We need to pop this out of the session so we can use it and delete it